src/main/resources/com/puppycrawl/tools/checkstyle/checks/modifier/messages_ja.properties

```diff
@@ -1,3 +1,4 @@
 annotation.order=''{0}'' 注釈修飾子は、非注釈修飾子に先行していません。
 mod.order=''{0}'' 修飾子が JLS 提案の順序に沿いません。
 redundantModifier=冗長な ''{0}'' 修飾子です。
+interface.implied.modifier=\u6697\u793A\u3055\u308C\u305F\u4FEE\u98FE\u5B50\u306F\u660E\u793A\u7684\u306B\u3059\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059: ''{0}''
```

Comment:
sure. I'd appreciate if you could give me  a few days to check it.
 I found that existing messages are not unicode-escaped but the added message is unicode-escaped. If not unicode-escaped message allowed to use, I think not unicode-escaped should be used because not unicode-escaped message makes maintenance easy.

Reply:
We need not unicode-escaped, just to be easy to read by others. Thanks a lot in advance.

Reply:
The original message is OK but the below message sounds more natural in Japanese.
暗黙的な ''{0}'' 修飾子は明示的に指定すべきです。

Reply:
Done